dotCMS/src/main/java/com/dotcms/aspects/aspectj/AspectJDelegateMethodInvocation.java

```diff
@@ -0,0 +1,55 @@
+package com.dotcms.aspects.aspectj;
+
+import com.dotcms.aspects.DelegateMethodInvocation;
+import com.dotcms.aspects.DotAspectException;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.Signature;
+import org.aspectj.lang.reflect.MethodSignature;
+
+import java.lang.reflect.Method;
+
+/**
+ * Implementation of delegate for Aspectj
+ * @author jsanca
+ */
+public class AspectJDelegateMethodInvocation<T> implements DelegateMethodInvocation<T> {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "static final long serialVersionUID" field to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2057)